doc/mvf_v4.rst

```diff
@@ -225,36 +225,42 @@ Calibration solutions
 
 Streams of type ``sdp.cal`` have the following keys.
 
-``antlist`` (list of string)
+``antlist`` (list of string, length ``n_ants``)
     List of antenna names. Arrays of calibration solutions use this
     order along the antenna axis.
 
-``pol_ordering`` (list of string)
+``pol_ordering`` (list of string, length ``n_pols``)
     List of polarisations (from ``v`` and ``h``). Arrays of calibration
     solutions use this order along the polarisation axis.
 
-``bls_ordering`` (2D array)
+``bls_ordering`` (list of string pairs, length ``n_bls``)
     Same meaning as for ``sdp.vis`` streams, but describes the internal
     ordering used within the calibration pipeline and not of much use to
     users.
```

Comment:
Interesting, I see sdp.vis bls_ordering is a numpy array while cal's bls_ordering is a list of lists of strings. Is that worth drawing attention to?

Reply:
Just one of the quirks of our namespace :-)

Should I change it to a "sequence of string pairs" to highlight the intended interface?

Reply:
Maybe. Or else just say that it's "either a list of string pairs or a 2D array" to make it clear that the type is uncertain.

Reply:
Done (for both instances of `bls_ordering`).